packages/build-info/package.json

```diff
@@ -6,7 +6,7 @@
   "exports": {
     ".": "./lib/index.js"
   },
-  "main": "./lib/index.js",
+  "browser": "./lib/index.js",
```

Comment:
we do not need an entry point for Node.js anymore?

Reply:
you can either have a `main` or `browser` field. browser just indicates it works for the browser but works on node the same. 

Furthermore on Node.js we just consume the cli entrypoint (inside buildbot) currently our only consumer.

After that we can change it and have the react-ui as second consumer

Reply:
I might be wrong, but looking through the docs of Node.js I do not think that node reads or evaluates the browser field. So assuming that I think right now the package won't work in Node.js.

The `browser` field was introduced by webpack in case a module has a separate entrypoint for browser-bundling  If not `browser` or `module` field then webpack will use `main`

I see this https://docs.npmjs.com/cli/v9/configuring-npm/package-json#browser though but we ship for both Node.js and Browser so I think we need both.

Reply:
I was interpreting this as either or https://docs.npmjs.com/cli/v9/configuring-npm/package-json#browser:

>  should be used instead of the main field.

Reply:
I know that is confusing. But I think what they mean is "If your project is not built for Node.js but only for the browser (like a react component) then only use the browser field."
But even that is kind of only half the truth, as there are these fields now `main`, `module`, `browser`, and `exports` which webpack or any bundler probably support in some order. :D

Reply:
Here is the algorithm of Node.js and there is no mention of `browser` only `main` and `exports` https://nodejs.org/api/modules.html#modules_all_together